mysql-test/mysql-test-run.pl

```diff
@@ -6796,6 +6819,21 @@ ($)
                         $opt_max_test_fail, set to 0 for no limit. Set
                         it's default with MTR_MAX_TEST_FAIL
 
+Environment variables controlling debugging parameters
+
+  MTR_TERM              Configures the terminal command to run the debugger.
```

Comment:
Comment alignment

Reply:
What's the issue with it? It's aligned as other parts of the help message.

Reply:
For example, I mean the alignment of lines "Configure..", "Defaults too .." , "can also".. doesn't seem to be aligned.

Reply:
Got it, some lines had tabs